src/main.rs

```diff
@@ -219,7 +220,7 @@ fn main() {
     let pool = models::establish_connection();
 
     let mut env_filter = EnvFilter::new(
-        "tokio_reactor=info,near=info,near=error,stats=info,telemetry=info,indexer_for_explorer=info,aggregated=info",
+        "tokio_reactor=info,near=info,near=error,stats=info,telemetry=info,indexer=info,indexer_for_explorer=info,aggregated=info",
```

Comment:
I can't find usages of this, do we need it?

What is the difference between "indexer" and "indexer_for_explorer"?

Reply:
`indexer` is NEAR Indexer (from nearcore) and `indexer_for_explorer` is current one